### DISCUSSION TITLE
UVTools currently affected by Windows Defender problems with Avalonia

### DISCUSSION THREAD

**Discussion by Thowil** [sn4k3/UVtools#212] [General]
I just wanted to  leave a note here that as of this writing UVTools 2.13.1 (using Avalonia 0.10.5) is affected by AvaloniaUI/Avalonia/issues/6009 and silently fails to start up on Windows with Defender enabled.
I hope this saves someone from a lengthy search...

**Comment by sn4k3** [comment]
Also here: https://github.com/sn4k3/UVtools/issues/202 and https://github.com/sn4k3/UVtools/issues/213
